src/lib/ngsi10/SubscribeContextRequest.h

```diff
@@ -54,6 +54,13 @@ typedef struct SubscribeContextRequest
   Throttling             throttling;             // Optional
   int64_t                expires;
 
+  struct {
```

Comment:
The example in data_model.md (check file name, I'm not sure) should be fixed, to put expression in the first level.